hummingbot/connector/exchange/huobi/huobi_user_stream_tracker.py

```diff
@@ -24,16 +21,15 @@ def logger(cls) -> HummingbotLogger:
         return cls._hust_logger
 
     def __init__(
-        self,
-        huobi_auth: Optional[HuobiAuth] = None,
-        api_factory: Optional[WebAssistantsFactory] = None,
+            self,
+            huobi_auth: Optional[HuobiAuth] = None,
+            api_factory: Optional[WebAssistantsFactory] = None,
     ):
-        super().__init__()
         self._huobi_auth: HuobiAuth = huobi_auth
-        self._ev_loop: asyncio.events.AbstractEventLoop = asyncio.get_event_loop()
-        self._data_source: Optional[UserStreamTrackerDataSource] = None
-        self._user_stream_tracking_task: Optional[asyncio.Task] = None
         self._api_factory = api_factory
+        super().__init__(data_source=HuobiAPIUserStreamDataSource(
```

Comment:
As discussed in the last stand up meeting, we need to do this in this case to instantiate the data source required by `super().__init__`. The class should be removed in future cleanups anyways, so we are accepting this situation for the moment, to avoid introducing big changes in all connectors with this PR intended for Kucoin.